src/functions-and-arrays.js

```diff
@@ -25,14 +73,57 @@ function sum() {}
 // Iteration #4: Calculate the average
 // Level 1: Array of numbers
 const numbersAvg = [2, 6, 9, 10, 7, 4, 1, 9];
+let avgNew = 0;
+
+function averageNumbers(arrAvg) {
+  
+  if(arrAvg.length === 0) {
+    return null
+  }
+  else if(arrAvg.length === 1) {
+    return arrAvg[0]
+  }
+  else {
+
+    let sum = 0
+    let average = 0
+    for(let i = 0; i < arrAvg.length; i++) {
+      sum += arrAvg[i]
+    }
+    average = sum / arrAvg.length
+    return average
+  }
 
-function averageNumbers() {}
+
+}
+
+averageNumbers(numbersAvg)
```

Comment:
you don't need to call your functions in this lab, Jasmine calls them for you. If you want to test, you can console.log them with a test array